packages/@aws-cdk/aws-appmesh/test/test.virtual-router.ts

```diff
@@ -244,6 +256,10 @@ export = {
           match: {
             prefixPath: '/path3',
           },
+          timeout: {
+            idle: cdk.Duration.seconds(11),
+            perRequest: cdk.Duration.seconds(11),
+          },
```

Comment:
No need for this (not used in an assertion).

Reply:
ack